addons/udes_stock/models/stock_move_line.py

```diff
@@ -173,6 +173,15 @@ def get_quants(self):
 
         return quants
 
+    def update_move_line(self, default_location_dest):
```

Comment:
We need a better name here.  `reinitialise_move_line` tells us more about the update that is taking place, but it is still a bit generic.  Ideally we want something that uses the language of the business domain.  How about `discard_accumulated_work` or similar?

Reply:
How about `reset_move_line_data`?

Reply:
Let's go with that then